src/library/features/crates/cratefeature.cpp

```diff
@@ -325,13 +353,15 @@ bool CrateFeature::activateCrate(CrateId crateId) {
     // TODO(XXX): How to select the newly created crate without
     // a corresponding table model? m_pCrateTableModel = nullptr
     // when creating crates by clicking the link on the HTML view.
-    if (m_pCrateTableModel) {
-        m_pCrateTableModel->selectCrate(crateId);
-        emit(showTrackModel(m_pCrateTableModel));
-    }
-    emit(enableCoverArtDisplay(true));
-    // Update selection
-    emit(featureSelect(this, m_lastRightClickedIndex));
+    // if (m_pCrateTableModel) {
```

Comment:
Same here, we need TODOs that explain what to do with the code that has been commented out.